README.md

```diff
@@ -18,6 +20,49 @@ Run `make build`.
+
+### Change Data Capture
+
+This connector implements CDC features for Vitess by connecting to a VStream that listens to changes in the configured table. Every detected change is converted into a record and returned in the call to `Read`. If there is no record available at the moment `Read` is called, it blocks until a record is available or the connector receives a stop signal.
+
+The connector in the CDC mode retrieves all the available shards from Vitess and tracks changes from all of them. If a reshard occurs, the connector will see the change and will listen for events from the new shards.
+
+### Position handling
+
+The connector goes through two modes.
+
+- Snapshot mode. The position contains `keyspace` and a value of the last processed element of an ordering column you chose. This means that the ordering column must contain unique values.
+
+- CDC mode. The position in this mode contains the same fields as in the Snapshot mode plus a list shards (and their unique shard transaction identifiers, gtid) of a `keyspace` you chose. The connector retrieves all the available shards at startup and watches them for changes.
+
+### Configuration Options
+
+| name             | description                                                                                                                                                                                  | required | default       |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------------- |
+| `address`        | The address pointed to a VTGate instance.<br />Format: `hostname:port`                                                                                                                       | **true** |               |
+| `table`          | The name of the table that the connector should write to.                                                                                                                                    | **true** |               |
+| `keyColumn`      | Column name used to detect if the target table already contains the record.                                                                                                                  | **true** |               |
+| `keyspace`       | The keyspace specifies a VTGate keyspace.                                                                                                                                                    | **true** |               |
+| `orderingColumn` | The name of a column that the connector will use for ordering rows.                                                                                                                          | **true** |               |
```

Comment:
please add  more information  about  this field. That we use this in position if it wrong choice will be problem with snapshot, maybe we can get some examples